default.nix

```diff
@@ -1,14 +1,40 @@
 {
   niv ? import nix/sources.nix,
-  pkgs ? import niv.nixpkgs { config = { allowUnfree = true; }; }
+  nixpkgs ? niv.nixpkgs,
+  pkgs ? import nixpkgs {}
 }:
+
 let
-  cbsLib = import ./lib { inherit pkgs; };
-  cbspkgs = import ./pkgs { inherit niv pkgs cbsLib; };
-  cbsModules = import ./modules { inherit niv cbspkgs pkgs; };
-in cbspkgs //
-  {
-    lib = cbsLib;
-    nixpkgs = pkgs;
-    modules = cbsModules;
-  }
+  lib = import ./lib { inherit pkgs; };
+  initrdSet = import ./pkgs/initrd-creator/release.nix {
+    inherit pkgs;
+    cbsLib = lib;
+  };
+  kernelSet = import ./pkgs/kernels { inherit pkgs; };
+in pkgs.recurseIntoAttrs rec {
+  inherit (niv) nixpkgs;
+  inherit lib;
+  modules = import ./modules { inherit niv; };
+
+  bender = pkgs.callPackage niv.bender {};
+
+  hydra = pkgs.callPackage ./pkgs/hydra { src = niv.hydra; };
+
+  initrds = pkgs.recurseIntoAttrs
+    (builtins.mapAttrs (_: pkgs.recurseIntoAttrs) initrdSet);
```

Comment:
Why does this change do?

Reply:
The `recurseIntoAttrs` function does this: https://github.com/NixOS/nixpkgs/blob/f75c11cfdfff3448623250e151cb47df70d4f224/lib/attrsets.nix#L482

With this field set, `nix-instantiate` and `nix-build` go deeper down the tree to find more derivations and build them if no specific attribute is set on the command line. This enables users to just run `nix-instantiate` to test if the whole repo is still evaluating nicely without errors. Otherwise the tools just stop on level 1 and don't go recursive (but hydra does because its evaluation strategy unfortunately differs from nix-instantiate).